.github/workflows/clang-format-linter.yml

```diff
@@ -7,6 +7,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@v4.2.2
```

Comment:
Pinning minor versions like this is not necessary.

Reply:
Missed reverting this one.  Sorry.  Will leave as is for now.